src/desktop/lan.desktop

```diff
@@ -6,12 +6,12 @@ X-KDE-ModuleType=Library
 X-KDE-HasReadOnlyMode=true
 X-SuSE-YaST-Call=lan
 
-X-SuSE-YaST-Group=Network
+X-SuSE-YaST-Group=System
 X-SuSE-YaST-Argument=
 X-SuSE-YaST-RootOnly=true
 X-SuSE-YaST-AutoInst=all
 X-SuSE-YaST-Geometry=
-X-SuSE-YaST-SortKey=
+X-SuSE-YaST-SortKey=!first
```

Comment:
What is the intent of this? In the qt control center it does not seem to have any effect.